packages/create-jest/src/runCreate.ts

```diff
@@ -7,10 +7,13 @@
 
 import * as path from 'path';
 import chalk = require('chalk');
+import exit = require('exit');
 import * as fs from 'graceful-fs';
 import prompts = require('prompts');
+import yargs = require('yargs');
+import {getVersion} from '@jest/core';
```

Comment:
This import will ship almost the whole Jest with `create-jest`. Not sure if that is worth.